packages/date-picker/test/keyboard-input.test.js

```diff
@@ -50,6 +51,7 @@ describe('keyboard', () => {
 
     it('should select focused date on Enter', async () => {
       await sendKeys({ type: '1/1/2001' });
+      await waitForOverlayRender();
```

Comment:
Updated all the tests using `sendKeys()` to make sure they wait for the overlay to fully open.